src/index.html

```diff
@@ -11,6 +11,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     action="http://localhost:3000/api"
+     method="post"
+    >
+      <label class="big-search">
+        <div
+          class="search-bar-big"
```

Comment:
According to this checklist part you can create common class `search-bar` and move all dublicate styles under it.
Add this class to both forms and modify styles only for small form using class-modifier `search-bar--small`

![image](https://user-images.githubusercontent.com/80700247/234277289-05a48662-cd57-4472-84b9-42606589d86c.png)

Fix all such cases